src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Options/EditorConfig/EditorConfigStorageLocation`1.cs

```diff
@@ -111,10 +111,10 @@ string IEditorConfigStorageLocation2.GetEditorConfigStringValue(object? value, O
             }
             else
             {
-                typedValue = (T)value;
+                typedValue = (T)value!;
```

Comment:
How do we know `value` isn't null here (seems like it could be)?

Reply:
This change just moves the `!` from the `return` statement to here.

---
In reply to: [518244029](https://github.com/dotnet/roslyn/pull/48803#discussion_r518244029) [](ancestors = 518244029)